### ISSUE TITLE
Output content differs between `--format tle` and `--format json`

### ISSUE THREAD

**Issue by lkatalin** [sigstore/rekor#2052]
When issuing a `rekor-cli get` command, the `--format` flag changes more than just the formatting of the output. It changes the actual content of what is output. I think these should output the same thing and/or there should be better documentation and flag names, ex. the flags could be `--basic` vs `--verbose` output or something similar.

Current output:

|field | present in --format tle  | present in --format json| same information displayed?|
| -------| ------------- | ------------- |----------|
| logIndex  | [x]  | [x] | [x] |
| logID | [x] | [x] | no |
| kindVersion | [x] | - | - |
| integratedTime | [x] | [x] | [x] |
| inclusionPromise | [x]| - | - |
| inclusionProof | [x] | - | - |
| canonicalizedBody | [x] | - | - |
| Body | - | [x] | - |
| UUID | - | [x] | - |

tl;dr only `logIndex` and `integratedTime` fields are consistent across output formats.

**Comment by bobcallaway** [comment]
* `logID` is base64 encoded in `tle` format, where as it is rendered as a hex string in `json` format
* the `UUID` can be computed from the `canonicalizedBody` in `tle` format
* `kindVersion` in `tle` format is visible within `Body` in `json`format

While I agree that this isn't the cleanest UX, `Body` in the `json` output is not really part of any standard within the sigstore ecosystem, but given that it has been an output format for a couple of years, we should probably be careful about changing it.
